draft-ietf-dnsop-svcb-https.md

```diff
@@ -750,7 +750,8 @@ pairs are concatenated to form the SvcParamValue.  These pairs MUST exactly
 fill the SvcParamValue; otherwise, the SvcParamValue is malformed.
 
 For "no-default-alpn", the presentation and wire format values MUST be
-empty.
+empty.  When "no-default-alpn" is specified, "alpn" MUST also be specified
+(else the record should be discarded as invalid).
```

Comment:
After thinking about this some more, I've changed my mind.  We can just say "this is not allowed" and leave the details to the implementors.
```suggestion
empty.  When "no-default-alpn" is specified, "alpn" MUST also be specified.
```

Reply:
With #285 this does cover things.  Although does that mean that "alpn" should be default mandatory but with a clarification that this doesn't mean clients need to understand all possible values?  As if "no-default-alpn" is mandatory and requires "alpn" for consistency then it would seem like "alpn" needs to be mandatory transitively?

Reply:
No, I don't think this means "alpn" needs to be "automatically mandatory".  Fundamentally, "automatically mandatory" mean "subtractive", i.e. "this key can subtract functionality from the endpoint".  "alpn" is purely additive, so it is always safe to ignore.